esp/src/eclwatch/ESPResult.js

```diff
@@ -160,8 +160,17 @@ define([
             var table = domConstruct.create("table", { border: 1, cellspacing: 0, width: "100%" }, node);
             if (lang.exists("Row", cell)) {
                 cell = cell.Row;
-            } 
-            if (Object.prototype.toString.call(cell) === '[object Array]') {
+            }
+            if (Object.prototype.toString.call(cell) === '[object Object]') {
+                //  Set of Strings  ---
```

Comment:
Set of [any scalar type] should really be supported.... it may be just this comment that is wrong.